app/components/post_draft/uploads/upload_item/upload_remove.test.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {fireEvent, renderWithEverything} from '@test/intl-test-helper';
+import TestHelper from '@test/test_helper';
+
+import UploadRemove from './upload_remove';
+
+import type {Database} from '@nozbe/watermelondb';
+
+jest.mock('@context/edit_post', () => ({
+    useEditPost: jest.fn(),
+}));
+
+const {useEditPost} = require('@context/edit_post');
+
+describe('UploadRemove', () => {
+    const serverUrl = 'serverUrl';
+    let database: Database;
+
+    beforeEach(async () => {
+        database = (await TestHelper.setupServerDatabase(serverUrl)!).database;
+    });
+
+    it('should call onFileRemove when isEditMode is true', () => {
```

Comment:
It would be nice to have a test for when isEditMode is false.

Reply:
Done.